core/templates/dev/head/css/oppia.css

```diff
@@ -1128,6 +1180,13 @@ md-card.oppia-gallery-tile {
   }
 }
 
+.oppia-alert-blocks {
```

Comment:
I do not know, this was what I reinstated from my previous push that somehow deleted this code.